docs/remote_svs.rst

```diff
@@ -0,0 +1,189 @@
+   while your team works remotely. If the office will be completely unattended,
+   consider storing the original SVS USB with senior staff or legal counsel.
+6. Prepare to respond to the loss or compromise of the remote SVS. At a
+   minimum, this would involve :ref:`rotating the Submission Key 
+   <rotate_submission_key>`, which would prevent an adversary from 
+   decrypting future submissions using the compromised key.
+
+Necessary Equipment
+-------------------
+
+In order to create a new SVS for remote use, you will need the following:
+
+* An air-gapped computer similar to the computer being used for your current
+  Secure Viewing Station. This workstation will be used for provisioning the
+  new SVS USB, and will also be used as part of the remote SVS system.
+      
+.. warning:: Any computer used as an SVS must be air-gapped by removing or
+             physically disabling all networking hardware (including
+             Bluetooth), and by removing or physically disabling speakers
+             and microphones. A computer used as an SVS should never be used
+             for any other purpose.
+    
+* An up-to-date Tails USB (the primary Tails USB). You do not need to set up
+  persistent storage on this device, as it will not be used during the SVS
+  setup process.
+* The current SVS USB, and its persistent volume’s passphrase
+* A USB key to act as the new SVS USB
+
+Creating New SVS USB Drives
+---------------------------
+
+To create the new SVS USB:
+
+1. Boot into Tails using the primary Tails USB on the air-gapped workstation.
+   When you see the welcome dialog, you can proceed without enabling persistence
+   or setting an admin password.
+2. Install Tails on the new SVS USB, following the instructions  
+   `here. <https://tails.boum.org/install/clone/pc/index.en.html>`_
+3. Boot into the new SVS USB and enable persistence with a strong passphrase
+   (a 6-word Diceware passphrase is recommended). In the Persistent volume
+   configuration wizard, be sure to enable persistence for “GnuPG - GnuPG
+   Keyrings and configuration”.
+4. Temporarily store the persistent volume passphrase in your password manager.
+   You should delete it once you have given the USB and passphrase to the
+   journalist who will be using them.
+5. Reboot the new SVS USB with persistence enabled and an administration
+   password set.
+6. Plug the current SVS USB into a free port on the workstation.
+7. Mount its persistent volume by browsing to Places > Computer, clicking
+   the USB disk in the left-hand column, and entering its persistent volume’s
+   passphrase.
+8. Open a terminal via Applications > Favorites > Terminal
+9. Copy the current SVS’s GPG keychain (which includes the Submission Key) to
+   the new SVS USB using the following command (without linebreaks):
+
+   ``sudo bash -c "rsync -a --no-specials --no-devices 
+   /media/amnesia/TailsData/gnupg/ /live/persistence/TailsData_unlocked/gnupg/"``
```

Comment:
This line overflows the right-hand margin -- maybe break the command up using `\`?